src/components/Header/WithImage.js

```diff
@@ -74,19 +74,19 @@ const ImageHeader = ({
           }}
           {...otherBackgroundImageProps}
           className={classNames({
-            "gradient--overlay": withOverlay,
-            "gradient--underlay": withUnderlay
+            "gradient--overlay": !withSpotLight && withOverlay,
+            "gradient--underlay": !withSpotLight && withUnderlay
           })}
           aria-hidden
         />
       ) : (
         <GradientBackground
-          from={from}
-          to={to}
+          stops={stops}
           deg={deg}
           className={classNames({
-            "gradient--overlay": withOverlay,
-            "gradient--underlay": withUnderlay
+            "gradient--overlay": !withSpotLight && withOverlay,
+            "gradient--underlay": !withSpotLight && withUnderlay,
+            "gradient--spotlight": withSpotLight
```

Comment:
Does `gradient--spotlight` break `gradient--overlay` and `gradient--underlay` when both classes are applied?

Reply:
It shouldn't but the spotlight gradient takes priority when it's applied & all other customizations are ignored.